### ISSUE TITLE
bash scripts: permission denied

### ISSUE THREAD

**Issue by xylar** [ACME-Climate/a-prime#17]
When I run a-prime, I'm getting a bunch of errors like:
```
./bash_scripts/aprime_atm_diags.bash: line 683: bash_scripts/plot_multiple_index_same_plot.bash: Permission denied
```
Indeed, when I look at execute permission on the bash scripts, many of them do not have it:
```
> ls -lah bash_scripts/
total 240K
drwxrwx--- 2 xylar xylar 4.0K Dec  1 00:16 .
drwxrwx--- 7 xylar xylar 4.0K Dec  1 00:36 ..
-rwxrwx--- 1 xylar xylar  33K Dec  1 00:16 aprime_atm_diags.bash
-rwxrwx--- 1 xylar xylar  781 Dec  1 00:16 aprime_ocnice_diags.bash
-rwxrwx--- 1 xylar xylar  958 Dec  1 00:16 batch_atm.lanl.bash
-rwxrwx--- 1 xylar xylar 1.1K Dec  1 00:16 batch_atm.nersc.bash
-rwxrwx--- 1 xylar xylar  681 Dec  1 00:16 batch_atm.olcf.bash
-rwxrwx--- 1 xylar xylar 1.2K Dec  1 00:16 batch_ocnice.lanl.bash
-rwxrwx--- 1 xylar xylar 1.3K Dec  1 00:16 batch_ocnice.nersc.bash
-rwxrwx--- 1 xylar xylar  922 Dec  1 00:16 batch_ocnice.olcf.bash
-rwxrwx--- 1 xylar xylar  630 Dec  1 00:16 batch_update_wwwdir.olcf.bash
-rwxrwx--- 1 xylar xylar 1.5K Dec  1 00:16 compute_climo.bash
-rw-rw---- 1 xylar xylar 1.5K Dec  1 00:16 compute_eqsoi_index.bash
-rw-rw---- 1 xylar xylar 1.8K Dec  1 00:16 compute_indices.bash
-rwxrwx--- 1 xylar xylar 1.1K Dec  1 00:16 condense_field.bash
-rwxrwx--- 1 xylar xylar 1.2K Dec  1 00:16 condense_field_bundle.bash
-rwxrwx--- 1 xylar xylar  27K Dec  1 00:16 generate_html_index_file.bash
-rwxrwx--- 1 xylar xylar 1.1K Dec  1 00:16 generate_unique_field_list.bash
-rw-rw---- 1 xylar xylar 1.2K Dec  1 00:16 generate_unique_group_list.bash
-rwxrwx--- 1 xylar xylar 2.6K Dec  1 00:16 plot_climo.bash
-rw-rw---- 1 xylar xylar 1.8K Dec  1 00:16 plot_enso_diags_time_series.bash
-rw-rw---- 1 xylar xylar 2.4K Dec  1 00:16 plot_enso_evolution.bash
-rw-rw---- 1 xylar xylar 2.8K Dec  1 00:16 plot_enso_feedbacks.bash
-rw-rw---- 1 xylar xylar 2.0K Dec  1 00:16 plot_enso_seasonality.bash
-rw-rw---- 1 xylar xylar 2.2K Dec  1 00:16 plot_multiple_index_same_plot.bash
-rw-rw---- 1 xylar xylar 2.7K Dec  1 00:16 plot_regr_nino34_fields.bash
-rw-rw---- 1 xylar xylar 2.0K Dec  1 00:16 plot_stddev.bash
-rwxrwx--- 1 xylar xylar 1.8K Dec  1 00:16 plot_time_series.bash
-rw-rw---- 1 xylar xylar 2.0K Dec  1 00:16 plot_tropical_pacific_meridional_avg.bash
-rwxrwx--- 1 xylar xylar 1.7K Dec  1 00:16 remap_climo_nco.bash
-rwxrwx--- 1 xylar xylar 1.4K Dec  1 00:16 remap_time_series_nco.bash
-rwxrwx--- 1 xylar xylar 4.0K Dec  1 00:16 setup.bash
-rwxrwx--- 1 xylar xylar 4.6K Dec  1 00:16 var_list_climo_model_vs_model.bash
-rwxrwx--- 1 xylar xylar 3.7K Dec  1 00:16 var_list_climo_model_vs_obs.bash
-rwxrwx--- 1 xylar xylar 1.6K Dec  1 00:16 var_list_enso_diags_bjerknes_feedback.bash
-rwxrwx--- 1 xylar xylar 2.2K Dec  1 00:16 var_list_enso_diags_climo.bash
-rwxrwx--- 1 xylar xylar  941 Dec  1 00:16 var_list_enso_diags_eqsoi_index.bash
-rwxrwx--- 1 xylar xylar 3.6K Dec  1 00:16 var_list_enso_diags_heat_flux-sst_feedbacks.bash
-rwxrwx--- 1 xylar xylar  922 Dec  1 00:16 var_list_enso_diags_nino_index.bash
-rwxrwx--- 1 xylar xylar 5.4K Dec  1 00:16 var_list_enso_diags_time_series.bash
-rwxrwx--- 1 xylar xylar 2.6K Dec  1 00:16 var_list_time_series_model_vs_model.bash
-rwxrwx--- 1 xylar xylar 2.6K Dec  1 00:16 var_list_time_series_model_vs_obs.bash
```
If I manually add execute permission, the errors seem to go away.

**Comment by milenaveneziani** [comment]
yes, this is what I was mentioning in #18

**Comment by milenaveneziani** [comment]
Fixed within #18.
